code/modules/clothing/spacesuits/rig/modules/modules.dm

```diff
@@ -63,7 +63,7 @@
 	var/activates_on_touch              // If set, unarmed attacks will call engage() on the target.
 	var/process_with_rig = TRUE			// If true, this calls process every tick when the rig does
 
-	var/active                          // Basic module status
+	var/active = 0                      // Basic module status
```

Comment:
this should probably be FALSE

although are you certain a null value isnt meaningful?

Reply:
It's not because if you will turn on and then turn off module var/active will be 0